guides/doc-Configuring_Virt-Who-VM-Subscriptions/master.adoc

```diff
@@ -0,0 +1,24 @@
+include::common/attributes.adoc[]
+include::common/header.adoc[]
+:context: virt-who-vm-subs
+:virt-who-vm-subs:
+:ProductName: {ProjectServer}
+
+= {ConfiguringVMSubscriptionsDocTitle}
```

Comment:
For some reason, the build displays "Configuring virtual machine subscriptions" as the title. I don't understand why.